client/src/actions/restaurants.ts

```diff
@@ -0,0 +1,7 @@
+import { createAction } from "typesafe-actions";
+import { PaginatedRestaurants } from "../models";
+import { LngLatBounds } from "mapbox-gl";
+
+export const restaurantsFetch = createAction("Restaurants Fetch")<[LngLatBounds | null, number]>();
```

Comment:
Should this be the following?
```suggestion
export const restaurantsFetch = createAction("Restaurants Fetch")<[LngLatBounds, number]>();
```